corehq/apps/users/models.py

```diff
@@ -1627,6 +1627,20 @@ def set_has_built_app(self):
             self.has_built_app = True
             self.save()
 
+    def log_user_create(self, created_by, created_via):
+        # fallback to self if not created by any user
+        created_by = created_by or self
+        log_model_change(
+            created_by,
+            self,
```

Comment:
I think this should be called with the django user:
```suggestion
            self.get_django_user(use_primary_db=True),
```

I think the created_by should also be the Django user.

Reply:
https://github.com/dimagi/commcare-hq/pull/27755/commits/4a74ea375d5b0ce8a4566e230025aa9a085a8414